From a35b4a2330ff042f362c8af69e6bbd3909a128d3 Mon Sep 17 00:00:00 2001
From: Vinicius Costa Gomes <vinicius.gomes@openbossa.org>
Date: Fri, 19 Feb 2010 11:34:55 -0300
Subject: [PATCH 1/1] Fix compilation when passing the --without-<driver> option

When we pass the --without-* option, the {withval} variable
was being set to "no".
---
 configure.ac |    5 ++---
 1 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index ba0f8a1..cca5ce9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -105,10 +105,9 @@ AC_ARG_ENABLE(debug, AC_HELP_STRING([--enable-debug],
 		CFLAGS="$CFLAGS -g"
 	fi
 ])
-
 phonebook_driver=dummy
 AC_ARG_WITH(phonebook, AC_HELP_STRING([--with-phonebook=DRIVER], [select phonebook driver]), [
-	phonebook_driver=${withval}
+	phonebook_driver=${withval}, phonebook_driver=dummy
 ])
 
 if (test "${phonebook_driver}" = "ebook"); then
@@ -129,7 +128,7 @@ AC_SUBST([PHONEBOOK_DRIVER], [phonebook-${phonebook_driver}.c])
 
 telephony_driver=dummy
 AC_ARG_WITH(telephony, AC_HELP_STRING([--with-telephony=DRIVER], [select telephony driver]), [
-	telephony_driver=${withval}
+	telephony_driver=${withval}, telephony_driver=dummy
 ])
 
 AC_SUBST([TELEPHONY_DRIVER], [telephony-${telephony_driver}.c])
-- 
1.6.2.5

